packages/admin/cms-admin/bin/comet-admin-generator.js

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
+require("ts-node").register({
+    require: ["tsconfig-paths/register"],
+});
```

Comment:
Why do you need this?

Reply:
needed to support config file as typescript (alternative ideas?)